### PR TITLE
fix: Correct the directing link of know more in about activity

### LINKED ISSUE #2031: Know more option in about  should direct to susi website.
**Describe the feature you'd like**
As we can see that the know more feature is directing towards a youtube video which is all about the feedback system and skill rating system,instead the feature should point to url where user can get more information about susi.

**Additional context**
In my opinion it should redirect the user to **https://dev.susi.ai/** so that user could a get a better opinion of susi.

**Screenshots**

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by chanmol1999**
Fixes #2031 

Changes: strings.xml

Screenshots for the change:

### CODE REVIEW

**Comment by chanmol1999** [comment]
Please review it @batbrain7

**Comment by chanmol1999** [comment]
Please review it @iamareebjamal @batbrain7

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -302,7 +302,7 @@
     <string name="susi_skill_cms">SUSI Skill CMS</string>
     <string name="url_susi_skill_cms">https://skills.susi.ai</string>
     <string name="susi_skill_cms_desc">SUSI is having many skills. You can look at the collection of skills at <a href="http://www.skills.susi.ai"><![CDATA[<u><font color='#4184f3'>skills.susi.ai</u></font>]]></a>. SUSI Skill development is easy and fun. You can edit existing skills or even create your own.</string>
-    <string name="susi_know_more_url" translatable="false">https://youtu.be/iOpA1mAy3QQ</string>
+    <string name="susi_know_more_url" translatable="false">https://dev.susi.ai</string>
```
Comment (iamareebjamal) [inline app/src/main/res/values/strings.xml]: Remove from strings and add as a string constant

**Comment by iamareebjamal** [comment]
Commit messages should be imperative. Correct, not Corrected

**Comment by chanmol1999** [comment]
Please merge it @iamareebjamal i'll make sure to follow commit message guidelines from next time.

**Comment by iamareebjamal** [comment]
You can still change the PR title

**Comment by chanmol1999** [comment]
Already i have change corrected to correct

**Pull request merged**